anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
The thing that might be confusing is that for multi cloze cards you make one note with a bunch of clozes in it.  When you add it to the deck it creates a batch of 'virtual' cards, one for each card number in the clozes.  c1, c2,... cn.  In reality every card is identical, it's use only changes at render time according to it's card number.